site/content/docs/5.3/layout/css-grid.md

```diff
@@ -227,13 +227,23 @@ Adding more rows and changing the placement of columns:
 
 ### Gaps
 
-Change the vertical gaps only by modifying the `row-gap`. Note that we use `gap` on `.grid`s, but `row-gap` and `column-gap` can be modified as needed.
+Note that we use `gap` on `.grid`s, but `row-gap` and `column-gap` can be modified as needed using either CSS properties, or CSS variables, or our [gap utilities]({{< docsref "/utilities/spacing#gap" >}}).
```

Comment:
It feels weird to start this section by "Note that [...]". It's like there's a missing explanation. Maybe you could rather have something like:
1. Change the gaps only by using our gap utilities + example
2. `Change the vertical gaps only by modifying the `row-gap`. Note that we use `gap` on `.grid`s, but `row-gap` and `column-gap` can be modified as needed.` + example
3. `Because of that, you can have different vertical and horizontal `gap`s, [...]` + example

What do you think?